mbed_app.json

```diff
@@ -1,5 +1,5 @@
 {
-    "requires": ["bare-metal", "sd", "filesystem", "fat_chan"],
+    
```

Comment:
Unfortunately this will break the datalogger (and possibly other LPC1549) builds. This should go into a separate repo.